packages/ui/src/common/components/typography/TokenValue.tsx

```diff
@@ -26,7 +27,6 @@ export const TokenValue = React.memo(({ className, value, size, isLoading }: Val
   if (value === null || !isDefined(value)) {
     return <span>-</span>
   }
-
   return (
     <ValueInJoys className={className} size={size}>
       {formatTokenValue(value)}
```

Comment:
Here I think we should enforce `value?: BN | null` it should never be a `number` to prevent any mistake.

So this would be:
```tsx
  if (!value) {
    return <span>-</span>
  }
  return (
    <ValueInJoys className={className} size={size}>
      {formatJoyValue(value)}
```